jac-uk/admin
# #1690 Update independent assessments UI

## What's included?
https://app.zenhub.com/files/201935567/942a1707-30b2-4c1e-b331-8dd7ee8670a6/download

Note: this PR is related to:
- Apply site: [#912 Select Assessor Type](https://github.com/jac-uk/apply/pull/911)
- Digital platform: [Update assessments functions](https://github.com/jac-uk/digital-platform/pull/761)
- Assessment site: [Decline assessment](https://github.com/jac-uk/assessments/pull/153)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to `Independent Assessments` of an exercise.
2. Click `Start Assessments` to start assessments.

    Not requested:
    - Check `Assessor type` column in the table.
    - Select assessments and click `Test requests` to send test assessment requests.
    - Select assessments and click `Send requests` to send assessment requests. And then the selected assessments should appear in `Requested` tab and `Sent` statistical figure should increase one.
    - Select assessments and click `Delete` and then the selected assessments should move to `Deleted` tab.

    https://user-images.githubusercontent.com/79906532/188426956-98d3a24f-563c-4c9f-9a70-7d8d9449fc2d.mov

    Requested:
    - Check `Assessor type` column in the table.
    - Select assessments and click `Test reminders` to send test assessment reminders.
    - Select assessments and click `Send reminders` to send assessment reminders.
    - Select assessments and click `Cancel`. It should be able to type in some reason. And then the selected assessments should move to `Cancelled` tab.
    - Select assessments and click `Reset`. And then the selected assessments should move to `Not requested` tab and `Sent` statistical figure should decrease one.
    - Select assessments and click `Delete` and then the selected assessments should move to `Deleted` tab.

    https://user-images.githubusercontent.com/79906532/188428271-8480e72b-8da4-43ce-8501-122f83ab4543.mov

    Completed:
    - Check `Assessor type` column in the table.
    - Select assessments and click `Cancel`. It should be able to type in some reason. And then the selected assessments should move to `Cancelled` tab and `Completed` statistical figure should decrease one..
    - Select assessments and click `Reset`. And then the selected assessments should move to `Not requested` tab and `Sent` and `Completed` statistical figures should decrease one.
    - Select assessments and click `Delete`. And then the selected assessments should move to `Requested` tab and `Completed` statistical figure should decrease one.
 
    Cancelled & Decline:
    - Check `Assessor type` column in the table.
    - Check the `Reason` column in the table.
    - Select assessments and click `Reset`. And then the selected assessments should move to `Not requested` tab and `Sent` statistical figure should decrease one.
 
    https://user-images.githubusercontent.com/79906532/188428499-03484677-8295-4352-88bc-2b9936d5dba4.mov

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
- No permission changes required

---
PREVIEW:DEVELOP
